images/build/debian-iptables/distroless/stage-binary-and-deps.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
This is based in https://github.com/kubernetes-sigs/kind/blob/ca68f7d4a2528e071c2969be956ca83fc1ed048f/images/haproxy/stage-binary-and-deps.sh